### DISCUSSION TITLE
Mobile menu viewport

### DISCUSSION THREAD

**Discussion by Przemko86** [holger1411/understrap#2050] [Q&A] [answered]
Hi, I'm looking for a place where I can change the screen width value from which the mobile menu appears.

Regards and thanks for help :)

**Comment by IanDelMar** [comment, marked as answer]
Use `.navbar-expand{-sm|-md|-lg|-xl|-xxl}` to determine when the navbar collapses.

Bootstrap 4:
https://github.com/understrap/understrap/blob/6a58dbbae4abb2ce4eaa1e315f0d5fe39e4acd50/global-templates/navbar-collapse-bootstrap4.php#L15

Bootstrap 5: 
https://github.com/understrap/understrap/blob/6a58dbbae4abb2ce4eaa1e315f0d5fe39e4acd50/global-templates/navbar-collapse-bootstrap5.php#L15

Bootstrap 5 Offcanvas:
https://github.com/understrap/understrap/blob/6a58dbbae4abb2ce4eaa1e315f0d5fe39e4acd50/global-templates/navbar-offcanvas-bootstrap5.php#L15

**Comment by Przemko86** [comment]
Cool! That's what I mean :) Thanks a lot. Merry Christmas :)
